worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -153,6 +154,13 @@ function routeTask(rootDocument) {
 
                 rootDocument.getField('CAF_WORKFLOW_ACTION').add(action.name);
                 applyActionDetails(rootDocument, actionDetails, terminateOnFailure);
+
+                if (action.applyMessagePrioritization && System.getenv("CAF_WMP_ENABLED")) {
```

Comment:
I'm approving but I wonder if we would be better to read the enabled flag from the settings service, so that we could control it on a per-tenant basis?  The settings service arguments should come through in the `args` just above.

Reply:
I have raised [US625155 ](https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=625155)to look into this.